mlflow/pyfunc/__init__.py

```diff
@@ -1171,15 +1169,18 @@ def udf(
                 # this prevents spark UDF task failing fast if other exception raised when scoring
                 # server launching.
                 _get_flavor_backend(
-                    local_model_path_on_executor, no_conda=False, install_mlflow=False
+                    local_model_path_on_executor, env_manager=EnvManager.LOCAL, install_mlflow=False
                 ).prepare_env(model_uri=local_model_path_on_executor, capture_output=True)
             else:
                 local_model_path_on_executor = local_model_path
             # launch scoring server
 
             # TODO: adjust timeout for server requests handler.
             scoring_server_proc = _get_flavor_backend(
-                local_model_path_on_executor, no_conda=False, workers=1, install_mlflow=False
+                local_model_path_on_executor,
+                env_manager=EnvManager.LOCAL,
```

Comment:
env_manager=EnvManager.CONDA

Reply:
I somehow misunderstood `no_conda=False` is equivalent to `EnvManager.LOCAL`.